ShellScripts/JavaOnTermux.sh

```diff
@@ -0,0 +1,25 @@
+#!/$PREFIX/bin/sh
```

Comment:
This line says to use `sh` shell to run this script. But when you will run this script as `./JavaOnTermux.sh` it will give error (bash syntax is different than sh) as it is meant to be run with `bash`. So one way is to replace this line to use bash or may remove it altogether to avoid the error.

As an improvement, the new installation of termux will need you to run `apt update` before `apt install` anything.

Reply:
Removed shebang and added "apt update" before "apt install".
Thanks for your suggestions.